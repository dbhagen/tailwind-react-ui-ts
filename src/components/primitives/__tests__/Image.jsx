import React from 'react'
import { shallow } from 'enzyme'

import { Image } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign({ src: '/static/assets/tubgirl.gif' }, testProps)

  const wrapper = shallow(<Image {...props} />)

  return {
    props,
    wrapper,
  }
}

describe('Image', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
