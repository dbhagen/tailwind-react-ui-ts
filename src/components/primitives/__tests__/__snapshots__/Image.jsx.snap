// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image renders matching snapshot 1`] = `
<Box
  inline={false}
  inlineBlock={false}
  is="div"
  relative={true}
  w="full"
>
  <Box
    bg="gray-light"
    inline={false}
    inlineBlock={false}
    is="div"
    style={
      Object {
        "paddingBottom": "100%",
      }
    }
  />
  <WithTheme(Base)
    absolute={true}
    inset={0}
    is="img"
    src="/static/assets/tubgirl.gif"
    w="full"
  />
</Box>
`;
